acceptance/acceptance_test.go

```diff
@@ -177,6 +177,16 @@ func getTests(t *testing.T) []string {
 	return testDirs
 }
 
+func setEnv(env []string, key, value string) []string {
+	for i, pair := range env {
+		if strings.HasPrefix(pair, key+"=") {
+			env[i] = key + "=" + value
```

Comment:
Is this necessary? What happens if you append duplicate key?

Reply:
No it's not. The last key is applicable otherwise. Modified this function to just append the duplicate key.